controller/src/main/java/gov/va/vro/controller/VroController.java

```diff
@@ -129,6 +129,15 @@ public ResponseEntity<FullHealthDataAssessmentResponse> postFullHealthAssessment
 
       FullHealthDataAssessmentResponse response =
           objectMapper.readValue(responseAsString, FullHealthDataAssessmentResponse.class);
+      if (response.getStatus().equals("ERROR")) {
+        log.info(
+            "Response from condition processor returned error message: {}",
+            response.getErrorMessage());
+        throw new ClaimProcessingException(
+            claim.getClaimSubmissionId(),
+            HttpStatus.INTERNAL_SERVER_ERROR,
+            "Error in condition processor.");
```

Comment:
Condition processor probably will not make much sense to caller. Maybe "Internal error during processing data for the claim.".